src/core/State.h

```diff
@@ -74,12 +76,12 @@ class State
     State();
     /**
      * Constructor
-     * @param contentWindows a list of contentWindows to serialize
+     * @param displayGroup the DisplayGroup to serialize
      */
-    State( const ContentWindowPtrs& contentWindows );
+    State( const DisplayGroupPtr displayGroup );
```

Comment:
remove const, this makes no sense for a smart pointer, and even less sense in a header.